From 0d70142cd35749802ba56c571fb3344688646905 Mon Sep 17 00:00:00 2001
From: James <howpigcanfly@outlook.com>
Date: Sun, 6 Aug 2017 19:42:26 +0800
Subject: [PATCH 3/3] [powerline-shell] Change default color

---
 themes/default.py | 24 ++++++++++++++++--------
 1 file changed, 16 insertions(+), 8 deletions(-)

diff --git a/themes/default.py b/themes/default.py
index 6526158..d9a0e41 100644
--- a/themes/default.py
+++ b/themes/default.py
@@ -12,16 +12,16 @@ class DefaultColor:
     USERNAME_BG = 240
     USERNAME_ROOT_BG = 124
 
-    HOSTNAME_FG = 250
-    HOSTNAME_BG = 238
+    HOSTNAME_FG = 0
+    HOSTNAME_BG = 148
 
     HOME_SPECIAL_DISPLAY = True
     HOME_BG = 31  # blueish
     HOME_FG = 15  # white
     PATH_BG = 237  # dark grey
     PATH_FG = 250  # light grey
-    CWD_FG = 254  # nearly-white grey
-    SEPARATOR_FG = 244
+    CWD_FG = 157  # nearly-white grey
+    SEPARATOR_FG = 251
 
     READONLY_BG = 124
     READONLY_FG = 254
@@ -34,13 +34,13 @@ class DefaultColor:
     REPO_DIRTY_BG = 161  # pink/red
     REPO_DIRTY_FG = 15  # white
 
-    JOBS_FG = 39
-    JOBS_BG = 238
+    JOBS_FG = 230
+    JOBS_BG = 166
 
     CMD_PASSED_BG = 236
     CMD_PASSED_FG = 15
-    CMD_FAILED_BG = 161
-    CMD_FAILED_FG = 15
+    CMD_FAILED_BG = 188
+    CMD_FAILED_FG = 16
 
     SVN_CHANGES_BG = 148
     SVN_CHANGES_FG = 22  # dark green
@@ -61,6 +61,14 @@ class DefaultColor:
     VIRTUAL_ENV_BG = 35  # a mid-tone green
     VIRTUAL_ENV_FG = 00
 
+    CMD_LINE_BG=237
+    CMD_LINE_FG=250
+
+
+    IDLE_FG = 199
+    IDLE_BG = 243
+
+
 class Color(DefaultColor):
     """
     This subclass is required when the user chooses to use 'default' theme.
-- 
1.8.3.1

