From f24211dddcaf6ee97eb32e24fee2566fd3a10e6b Mon Sep 17 00:00:00 2001
From: Li jin <lijin@hgrica.com>
Date: Sat, 3 Sep 2016 13:27:18 +0800
Subject: [PATCH] powerline-shell change idle color

---
 segments/idle.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/segments/idle.py b/segments/idle.py
index f110948..49c429e 100644
--- a/segments/idle.py
+++ b/segments/idle.py
@@ -4,6 +4,8 @@ def add_idle_segment(powerline):
         'zsh': ' \n ',
         'bare': ' \n ',
     }
-    fg = Color.CMD_LINE_FG
-    bg = Color.CMD_LINE_BG
+    #  fg = Color.CMD_LINE_FG
+    #  bg = Color.CMD_LINE_BG
+    fg = Color.HOME_BG
+    bg = Color.HOME_FG
     powerline.append(idle_indicators[powerline.args.shell], fg, bg)
-- 
2.1.4

