From 2745c33d6ed905f2f3a8e33fafaeeae8d3fc2cf8 Mon Sep 17 00:00:00 2001
From: Li jin <lijin@hgrica.com>
Date: Sat, 3 Sep 2016 13:27:18 +0800
Subject: [PATCH 6/7] powerline-shell change idle color

---
 segments/idle.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/segments/idle.py b/segments/idle.py
index f110948..49c429e 100644
--- a/segments/idle.py
+++ b/segments/idle.py
@@ -4,6 +4,8 @@ def add_idle_segment(powerline):
         'zsh': ' \n ',
         'bare': ' \n ',
     }
-    fg = Color.CMD_LINE_FG
-    bg = Color.CMD_LINE_BG
+    #  fg = Color.CMD_LINE_FG
+    #  bg = Color.CMD_LINE_BG
+    fg = Color.HOME_BG
+    bg = Color.HOME_FG
     powerline.append(idle_indicators[powerline.args.shell], fg, bg)
-- 
1.8.3.1

