From 3c413eef4ce2b923d3ca7b04316ebaa5e00c144e Mon Sep 17 00:00:00 2001
From: "jin.li" <jin.li@nufront.com>
Date: Sun, 25 Jun 2017 23:26:30 +0800
Subject: [PATCH] powerline-shell update color

---
 themes/default.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/themes/default.py b/themes/default.py
index 9e498f5..24249e4 100644
--- a/themes/default.py
+++ b/themes/default.py
@@ -20,7 +20,7 @@ class DefaultColor:
     HOME_FG = 15  # white
     PATH_BG = 13  # dark grey
     PATH_FG = 16  # light grey
-    CWD_FG = 254  # nearly-white grey
+    CWD_FG = 157  # nearly-white grey
     SEPARATOR_FG = 237
 
     READONLY_BG = 124
-- 
1.8.3.1

