diff --git a/config.py.dist b/config.py.dist
index a1e7627..0b841d2 100644
--- a/config.py.dist
+++ b/config.py.dist
@@ -13,12 +13,12 @@ SEGMENTS = [
 # Show current virtual environment (see http://www.virtualenv.org/)
     'virtual_env',
 
-# Show the current user's username as in ordinary prompts
-    'username',
-
 # Show the machine's hostname. Mostly used when ssh-ing into other machines
     'hostname',
 
+# Show the current user's username as in ordinary prompts
+    'username',
+
 # Show a padlock when ssh-ing from another machine
     'ssh',
 
@@ -26,12 +26,17 @@ SEGMENTS = [
 # replaced with ellipsis ('...')
     'cwd',
 
+# Shows a '#' if the current user is root, '$' otherwise
+# Also, changes color if the last command exited with a non-zero error code
+    'root',
+
 # Show a padlock if the current user has no write access to the current
 # directory
     'read_only',
 
 # Show the current git branch and status
-    'git',
+	# 'git',
+	'git_ext',
 
 # Show the current mercurial branch and status
     'hg',
@@ -42,15 +47,15 @@ SEGMENTS = [
 # Show the current fossil branch and status
     'fossil',
 
+# Show the last command's exit code if it was non-zero
+#    'exit_code',
+
 # Show number of running jobs
     'jobs',
 
-# Show the last command's exit code if it was non-zero
-#    'exit_code',
+# Add new line in command window
+	'idle',
 
-# Shows a '#' if the current user is root, '$' otherwise
-# Also, changes color if the last command exited with a non-zero error code
-    'root',
 ]
 
 # Change the colors used to draw individual segments in your prompt
diff --git a/segments/android_prj.py b/segments/android_prj.py
new file mode 100644
index 0000000..24d38e2
--- /dev/null
+++ b/segments/android_prj.py
@@ -0,0 +1,28 @@
+
+import os
+import subprocess
+import string
+import sys
+
+def add_android_prj_segment(powerline):
+	get_info = subprocess.Popen(["/home/$WHOAMI/.vim/bundle/powerline-shell/segments/get_android_prj.sh",""], stdout=subprocess.PIPE)
+	branch = get_info.stdout.read()
+	#  g_st = string.find(branch,"*")
+
+        #  get_tmp=subprocess.Popen('/home/$WHOAMI/1.sh',stdout=subprocess.PIPE,shell=True)
+        #  print(get_tmp.stdout.readlines())
+        #  out,err = p.communicate()
+        #  print out
+        #  print err
+
+
+	fg = Color.REPO_CLEAN_FG
+	bg = Color.REPO_CLEAN_BG
+	#  if g_st >= 0:
+	#          ## can not find * in branch
+	#          fg = Color.REPO_DIRTY_FG
+	#          bg = Color.REPO_DIRTY_BG
+	s_len = len(branch)
+	if s_len > 0:
+		powerline.append(' %s ' % branch, fg, bg)
+
diff --git a/segments/get_android_prj.sh b/segments/get_android_prj.sh
new file mode 100644
index 0000000..70aa52b
--- /dev/null
+++ b/segments/get_android_prj.sh
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+
+get_pwd=$(pwd)
+
+# echo "get produce: $TARGET_PRODUCT"
+
+if [ "$TARGET_PRODUCT" != "" ]
+then
+	echo -n "$TARGET_PRODUCT-$TARGET_BUILD_VARIANT"
+	# source ~/.vim/bundle/powerline-shell/segments/android_info.sh
+fi
+
+
diff --git a/segments/get_git_info.sh b/segments/get_git_info.sh
new file mode 100755
index 0000000..3c9ee5e
--- /dev/null
+++ b/segments/get_git_info.sh
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ -f "/etc/bash_completion.d/git-prompt" ]
+then
+	source /etc/bash_completion.d/git-prompt
+	source ~/.vim/bundle/powerline-shell/segments/info.sh
+else
+	if [ -f "/etc/bash_completion.d/git" ]
+	then
+		source /etc/bash_completion.d/git
+		source /usr/share/git-core/contrib/completion/git-prompt.sh
+		source ~/.vim/bundle/powerline-shell/segments/info.sh
+	fi
+fi
diff --git a/segments/git_ext.py b/segments/git_ext.py
new file mode 100644
index 0000000..8a378ab
--- /dev/null
+++ b/segments/git_ext.py
@@ -0,0 +1,21 @@
+
+import os
+import subprocess
+import string
+import sys
+
+def add_git_ext_segment(powerline):
+	#  get_info = subprocess.Popen(["/home/$WHOAMI/.vim/bundle/powerline-shell/segments/get_git_info.sh",""], stdout=subprocess.PIPE)
+	#  branch = get_info.stdout.read()
+        branch = ""
+	g_st = string.find(branch,"*")
+	fg = Color.REPO_CLEAN_FG
+	bg = Color.REPO_CLEAN_BG
+	if g_st >= 0:
+		## can not find * in branch
+		fg = Color.REPO_DIRTY_FG
+		bg = Color.REPO_DIRTY_BG
+	s_len = len(branch)
+	if s_len > 0:
+		powerline.append(' %s ' % branch, fg, bg)
+
diff --git a/segments/idle.py b/segments/idle.py
new file mode 100644
index 0000000..f110948
--- /dev/null
+++ b/segments/idle.py
@@ -0,0 +1,9 @@
+def add_idle_segment(powerline):
+    idle_indicators = {
+        'bash': '\n ',
+        'zsh': ' \n ',
+        'bare': ' \n ',
+    }
+    fg = Color.CMD_LINE_FG
+    bg = Color.CMD_LINE_BG
+    powerline.append(idle_indicators[powerline.args.shell], fg, bg)
diff --git a/segments/info.sh b/segments/info.sh
new file mode 100644
index 0000000..2091b6a
--- /dev/null
+++ b/segments/info.sh
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+__git_ps1
+
+
diff --git a/themes/default.py b/themes/default.py
index 08b1ff4..f7ccd8c 100644
--- a/themes/default.py
+++ b/themes/default.py
@@ -3,20 +3,20 @@ class DefaultColor:
     This class should have the default colors for every segment.
     Please test every new segment with this theme first.
     """
-    USERNAME_FG = 250
-    USERNAME_BG = 240
+    USERNAME_FG = 254
+    USERNAME_BG = 33
     USERNAME_ROOT_BG = 124
 
-    HOSTNAME_FG = 250
-    HOSTNAME_BG = 238
+    HOSTNAME_FG = 0
+    HOSTNAME_BG = 148
 
     HOME_SPECIAL_DISPLAY = True
     HOME_BG = 31  # blueish
     HOME_FG = 15  # white
-    PATH_BG = 237  # dark grey
-    PATH_FG = 250  # light grey
+    PATH_BG = 13  # dark grey
+    PATH_FG = 16  # light grey
     CWD_FG = 254  # nearly-white grey
-    SEPARATOR_FG = 244
+    SEPARATOR_FG = 237
 
     READONLY_BG = 124
     READONLY_FG = 254
@@ -29,8 +29,8 @@ class DefaultColor:
     REPO_DIRTY_BG = 161  # pink/red
     REPO_DIRTY_FG = 15  # white
 
-    JOBS_FG = 39
-    JOBS_BG = 238
+    JOBS_FG = 230
+    JOBS_BG = 166
 
     CMD_PASSED_BG = 236
     CMD_PASSED_FG = 15
@@ -56,6 +56,9 @@ class DefaultColor:
     VIRTUAL_ENV_BG = 35  # a mid-tone green
     VIRTUAL_ENV_FG = 00
 
+    CMD_LINE_BG=237
+    CMD_LINE_FG=250
+
 class Color(DefaultColor):
     """
     This subclass is required when the user chooses to use 'default' theme.
