From f4c586caf375d3553ffcc0bc078a5f46327d6aa5 Mon Sep 17 00:00:00 2001
From: James <howpigcanfly@outlook.com>
Date: Sun, 6 Aug 2017 13:01:00 +0800
Subject: [PATCH 12/12] [powerline-shell] Add fail color to idle segment

---
 segments/idle.py | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/segments/idle.py b/segments/idle.py
index 523fece..ca3d693 100644
--- a/segments/idle.py
+++ b/segments/idle.py
@@ -6,6 +6,9 @@ def add_idle_segment(powerline):
     #  }
     #  fg = Color.CMD_LINE_FG
     #  bg = Color.CMD_LINE_BG
-    fg = Color.HOME_BG
-    bg = Color.HOME_FG
+    fg = Color.IDLE_FG
+    bg = Color.IDLE_BG
+    if powerline.args.prev_error != 0:
+        fg = Color.CMD_FAILED_FG
+        bg = Color.CMD_FAILED_BG
     powerline.append("", fg, bg)
-- 
1.8.3.1

