diff --git a/config.py.dist b/config.py.dist
index e810d80..f7bfe3f 100644
--- a/config.py.dist
+++ b/config.py.dist
@@ -31,7 +31,8 @@ SEGMENTS = [
     'read_only',
 
 # Show the current git branch and status
-    'git',
+	# 'git',
+	'ext_git',
 
 # Show the current mercurial branch and status
     'hg',
diff --git a/segments/ext_git.py b/segments/ext_git.py
new file mode 100644
index 0000000..4647601
--- /dev/null
+++ b/segments/ext_git.py
@@ -0,0 +1,14 @@
+
+import os
+import subprocess
+
+def add_ext_git_segment(powerline):
+    get_info = subprocess.Popen(["/home/$WHOAMI/.vim/bundle/powerline-shell/segments/get_git_info.sh",""], stdout=subprocess.PIPE)
+    branch = get_info.stdout.read()
+    fg = Color.REPO_DIRTY_FG
+    bg = Color.REPO_DIRTY_BG
+    powerline.append(' %s ' % branch, fg, bg)
+
+
+
+
diff --git a/segments/get_git_info.sh b/segments/get_git_info.sh
new file mode 100755
index 0000000..6fc8eb9
--- /dev/null
+++ b/segments/get_git_info.sh
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+source /etc/bash_completion.d/git-prompt;source ~/.vim/bundle/powerline-shell/segments/info.sh
diff --git a/segments/info.sh b/segments/info.sh
new file mode 100755
index 0000000..2091b6a
--- /dev/null
+++ b/segments/info.sh
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+__git_ps1
+
+
