diff --git a/config.py.dist b/config.py.dist
index a1e7627..e810d80 100644
--- a/config.py.dist
+++ b/config.py.dist
@@ -13,12 +13,12 @@ SEGMENTS = [
 # Show current virtual environment (see http://www.virtualenv.org/)
     'virtual_env',
 
-# Show the current user's username as in ordinary prompts
-    'username',
-
 # Show the machine's hostname. Mostly used when ssh-ing into other machines
     'hostname',
 
+# Show the current user's username as in ordinary prompts
+    'username',
+
 # Show a padlock when ssh-ing from another machine
     'ssh',
 
@@ -42,15 +42,19 @@ SEGMENTS = [
 # Show the current fossil branch and status
     'fossil',
 
-# Show number of running jobs
-    'jobs',
-
 # Show the last command's exit code if it was non-zero
 #    'exit_code',
 
 # Shows a '#' if the current user is root, '$' otherwise
 # Also, changes color if the last command exited with a non-zero error code
     'root',
+
+# Show number of running jobs
+    'jobs',
+
+# Add new line in command window
+	'idle',
+
 ]
 
 # Change the colors used to draw individual segments in your prompt
diff --git a/segments/idle.py b/segments/idle.py
new file mode 100644
index 0000000..f110948
--- /dev/null
+++ b/segments/idle.py
@@ -0,0 +1,9 @@
+def add_idle_segment(powerline):
+    idle_indicators = {
+        'bash': '\n ',
+        'zsh': ' \n ',
+        'bare': ' \n ',
+    }
+    fg = Color.CMD_LINE_FG
+    bg = Color.CMD_LINE_BG
+    powerline.append(idle_indicators[powerline.args.shell], fg, bg)
diff --git a/themes/default.py b/themes/default.py
index 08b1ff4..f7ccd8c 100644
--- a/themes/default.py
+++ b/themes/default.py
@@ -3,20 +3,20 @@ class DefaultColor:
     This class should have the default colors for every segment.
     Please test every new segment with this theme first.
     """
-    USERNAME_FG = 250
-    USERNAME_BG = 240
+    USERNAME_FG = 254
+    USERNAME_BG = 33
     USERNAME_ROOT_BG = 124
 
-    HOSTNAME_FG = 250
-    HOSTNAME_BG = 238
+    HOSTNAME_FG = 0
+    HOSTNAME_BG = 148
 
     HOME_SPECIAL_DISPLAY = True
     HOME_BG = 31  # blueish
     HOME_FG = 15  # white
-    PATH_BG = 237  # dark grey
-    PATH_FG = 250  # light grey
+    PATH_BG = 13  # dark grey
+    PATH_FG = 16  # light grey
     CWD_FG = 254  # nearly-white grey
-    SEPARATOR_FG = 244
+    SEPARATOR_FG = 237
 
     READONLY_BG = 124
     READONLY_FG = 254
@@ -29,8 +29,8 @@ class DefaultColor:
     REPO_DIRTY_BG = 161  # pink/red
     REPO_DIRTY_FG = 15  # white
 
-    JOBS_FG = 39
-    JOBS_BG = 238
+    JOBS_FG = 230
+    JOBS_BG = 166
 
     CMD_PASSED_BG = 236
     CMD_PASSED_FG = 15
@@ -56,6 +56,9 @@ class DefaultColor:
     VIRTUAL_ENV_BG = 35  # a mid-tone green
     VIRTUAL_ENV_FG = 00
 
+    CMD_LINE_BG=237
+    CMD_LINE_FG=250
+
 class Color(DefaultColor):
     """
     This subclass is required when the user chooses to use 'default' theme.
