diff --git a/tools/cmake/Makefile b/tools/cmake/Makefile
index 888cf7de8e..cd676888b1 100644
--- a/tools/cmake/Makefile
+++ b/tools/cmake/Makefile
@@ -20,13 +20,24 @@ HOST_CONFIGURE_PARALLEL:=1
 
 include $(INCLUDE_DIR)/host-build.mk
 
+
+HOST_ROOTFS=/home/lijin/xprojects/openwrt/staging_dir/host
+EXTERN_ROOTFS=/home/lijin/Environment/env_rootfs
+
+# HOST_CFLAGS += -DCMAKE_C_COMPILER=$(HOST_ROOTFS)/bin/gcc
+# HOST_CFLAGS += -DCMAKE_CXX_COMPILER=$(HOST_ROOTFS)/bin/g++
+HOST_CFLAGS += -DCMAKE_C_FLAGS=-I$(EXTERN_ROOTFS)/include
+HOST_CFLAGS += -DCMAKE_C_LINK_FLAGS=-L$(EXTERN_ROOTFS)/lib64 -Wl,-rpath=$(EXTERN_ROOTFS)/lib64 -L$(EXTERN_ROOTFS)/lib -Wl,-rpath=$(EXTERN_ROOTFS)/lib -L$(EXTERN_ROOTFS)/lib64 -Wl,-rpath=$(EXTERN_ROOTFS)/lib64 -L$(EXTERN_ROOTFS)/lib -Wl,-rpath=$(EXTERN_ROOTFS)/lib
+HOST_CFLAGS += -DCMAKE_CXX_LINK_FLAGS=-L$(EXTERN_ROOTFS)/lib64 -Wl,-rpath=$(EXTERN_ROOTFS)/lib64 -L$(EXTERN_ROOTFS)/lib -Wl,-rpath=$(EXTERN_ROOTFS)/lib -L$(EXTERN_ROOTFS)/lib64 -Wl,-rpath=$(EXTERN_ROOTFS)/lib64 -L$(EXTERN_ROOTFS)/lib -Wl,-rpath=$(EXTERN_ROOTFS)/lib
+HOST_CFLAGS += -DCMAKE_EXE_LINKER_FLAGS=-static-libgcc -static-libstdc++
+
 HOST_CONFIGURE_VARS += \
 	MAKEFLAGS="$(HOST_JOBS)" \
-	CXXFLAGS="$(HOST_CFLAGS)"
+	CXXFLAGS='$(HOST_CFLAGS)'
 
 HOST_CONFIGURE_ARGS := \
 	$(if $(MAKE_JOBSERVER),--parallel="$(MAKE_JOBSERVER)") \
-	--prefix=$(STAGING_DIR_HOST)
+	--prefix=$(STAGING_DIR_HOST) \
 
 ifneq ($(findstring c,$(OPENWRT_VERBOSE)),)
   HOST_MAKE_FLAGS += VERBOSE=1
